### ISSUE TITLE
[Bug] Spleeter 1.5.0 installs Tensorflow 1.14.0 through Conda

### ISSUE THREAD

**Issue by Juan-Carlos-Rodero-Sales-Bose** [deezer/spleeter#326]
## Description
Downloaded `spleeter` 1.5.0 from `conda` and it installs the wrong tensorflow version.

## Step to reproduce

1. Installed using `conda install -c conda-forge spleeter`

## Output

```
$> conda search spleeter=1.5.0=py37hc8dfbb8_0 --info
Loading channels: done
spleeter 1.5.0 py37hc8dfbb8_0
-----------------------------
file name   : spleeter-1.5.0-py37hc8dfbb8_0.tar.bz2
name        : spleeter
version     : 1.5.0
build       : py37hc8dfbb8_0
build number: 0
size        : 64 KB
license     : MIT
subdir      : linux-64
url         : https://conda.anaconda.org/conda-forge/linux-64/spleeter-1.5.0-py37hc8dfbb8_0.tar.bz2
md5         : 41d09e54e42fa2f56a0285021d720f05
timestamp   : 2020-03-20 22:05:14 UTC
dependencies: 
  - ffmpeg-python
  - librosa 0.7.2
  - norbert
  - pandas 0.25.1
  - python >=3.7,<3.8.0a0
  - python_abi 3.7.* *_cp37m
  - requests
  - setuptools
  - tensorflow 1.14.0
```

## Environment

|                   |                                 |
| ----------------- | ------------------------------- |
| OS                |  Linux  |
| Installation type | Conda          |
| RAM available     | 32Gb                           |
| Hardware spec     | CPU: Intel Xeon Platinum 8000           |

## Additional context

Ran into disparity between the `requirements.txt` and the actual dependencies installed by `conda`

**Comment by mmoussallam** [comment]
Hi @Juan-Carlos-Rodero-Sales-Bose 

Thanks for raising the issue we'll look into this, it's indeed quite strange.

**Comment by Faylixe** [comment]
This will be fixed by 1.5.1 release (currently in deployment on Conda-forge).

**Comment by Juan-Carlos-Rodero-Sales-Bose** [comment]
Thank you guys!
